ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-general.j2

```diff
@@ -20,10 +20,13 @@ router general
 {%     endif %}
 {%     for vrf in router_general.vrfs | arista.avd.natural_sort %}
    vrf {{ vrf }}
-{%         for leak_route in router_general.vrfs[vrf].leak_routes | arista.avd.natural_sort %}
+{%         for leak_route in router_general.vrfs[vrf].leak_routes | arista.avd.natural_sort if router_general.vrfs[vrf].leak_routes is arista.avd.defined %}
```

Comment:
We need to wait for https://github.com/aristanetworks/ansible-avd/pull/1664 to get merged into devel because we are updating the data-model. You can use the new data-model once it is merged.

Reply:
I reverted this change so we are not waiting on #1664

Reply:
Even then we will have issues. It wont take time to merge that one. The problem was with using ```router_general.vrfs[vrf].routes.dynamic_prefix_lists``` as this will be changed to ```vrf.routes.dynamic_prefix_lists```. But you can't use that before the model isn't updated.